tools/generate_xml_monsters_rme.py

```diff
@@ -0,0 +1,115 @@
+                    'looklegs': outfit.get('lookLegs', '0'),
+                    'lookfeet': outfit.get('lookFeet', '0'),
+                    'lookaddons': outfit.get('lookAddons', '0')
+                })
+                # Remover atributos que são '0' (exceto looktype)
+                attributes = {k: v for k, v in attributes.items() if v != '0' or k == 'looktype'}
+            else:
+                # Caso contrário, incluir somente looktype se existir
+                if 'lookType' in outfit:
+                    attributes['looktype'] = outfit['lookType']
+        
+        # Criar o elemento XML
+        monster_elem = etree.SubElement(root, "monster", **attributes)
+    
+    # Criar a árvore XML e adicionar a declaração XML
+    tree = etree.ElementTree(root)
+    with open(output_file, 'wb') as f:
+        f.write(b'<?xml version="1.0" encoding="UTF-8"?>\n')
+        tree.write(f, encoding='utf-8', xml_declaration=False, pretty_print=True)
+    
+    # Adicionar os arquivos não processados no final do arquivo XML
+    if skipped_files:
+        skipped_elem = etree.SubElement(root, "skipped_files")
+        for file in skipped_files:
+            etree.SubElement(skipped_elem, "file").text = file
+
+# Função principal que percorre as pastas e processa os arquivos Lua
+def process_lua_files(root_folder, output_xml):
+    monsters_data = []
+    skipped_files = []
+    
+    for subdir, _, files in os.walk(root_folder):
+        for file in files:
+            if file.endswith('.lua'):
+                file_path = os.path.join(subdir, file)
+                with open(file_path, 'r', encoding='utf-8') as f:
+                    content = f.read()
+                    name = extract_monster_name(content)
+                    outfit = extract_monster_outfit(content)
+                    if name and outfit:
+                        # Verificar se o monstro atende às condições para ser incluído
+                        has_non_zero_outfit = any(
+                            outfit.get(key, '0') != '0'
+                            for key in ['lookHead', 'lookBody', 'lookLegs', 'lookFeet', 'lookAddons']
+                        )
+                        
+                        if ('lookTypeEx' in outfit and outfit['lookTypeEx'] != '0') or has_non_zero_outfit or 'lookType' in outfit:
+                            monsters_data.append({'name': name, 'outfit': outfit})
+                        else:
+                            skipped_files.append(file_path)
+                    else:
+                        skipped_files.append(file_path)
+    
+    generate_xml(monsters_data, output_xml, skipped_files)
+
+# Chamada da função principal com o diretório de entrada e arquivo XML de saída
+root_folder = 'C:/Users/Usuario/Documents/GitHub/fork-canary/data-otservbr-global/monster'
+output_xml = 'monstros.xml'
+process_lua_files(root_folder, output_xml)
+
+print(f"Arquivo XML gerado com sucesso: {output_xml}")
```

Comment:
Please add this verification to avoid running this method when being imported.
```suggestion
if __name__ == '__main__':
    process_lua_files(root_folder, output_xml)

    print(f"Arquivo XML gerado com sucesso: {output_xml}")
```